SPECS/cairo/cairo.spec

```diff
@@ -1,46 +1,49 @@
 Summary:        A 2D graphics library.
 Name:           cairo
-Version:        1.17.4
-Release:        3%{?dist}
+Version:        1.18.0
+Release:        1%{?dist}
 License:        LGPLv2 OR MPLv1.1
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Libraries
 URL:            https://cairographics.org
-Source0:        https://cairographics.org/snapshots/%{name}-%{version}.tar.xz
+Source0:        https://cairographics.org/releases/%{name}-%{version}.tar.xz
+
 
-BuildRequires:  fontconfig-devel
-BuildRequires:  freetype-devel
 BuildRequires:  gcc
-BuildRequires:  glib-devel
-BuildRequires:  libpng-devel
-BuildRequires:  libX11-devel
-BuildRequires:  libxml2-devel
-BuildRequires:  libXrender-devel
-BuildRequires:  pixman-devel
-BuildRequires:  pkg-config
+BuildRequires:  gcc-c++
+BuildRequires:  gtk-doc
+BuildRequires:  meson
+BuildRequires:  pkgconfig(expat)
+BuildRequires:  pkgconfig(pixman-1)
+BuildRequires:  pkgconfig(freetype2)
+BuildRequires:  pkgconfig(fontconfig)
+BuildRequires:  pkgconfig(gobject-2.0)
+BuildRequires:  pkgconfig(libpng)
 BuildRequires:  pkgconfig(xext)
-
-Requires:       expat
-Requires:       glib
-Requires:       libpng
-Requires:       pixman
+BuildRequires:  pkgconfig(xcb-render)
+BuildRequires:  pkgconfig(xrender)
 
 %description
-Cairo is a 2D graphics library with support for multiple output devices.
-
+Cairo is a 2D graphics library designed to provide high-quality display
+and print output. Currently supported output targets include the X Window
+System, in-memory image buffers, and image files (PDF, PostScript, and SVG).
+ 
+Cairo is designed to produce consistent output on all output media while
+taking advantage of display hardware acceleration when available.
+ 
 %package        devel
-Summary:        Header and development files
+Summary:        Development files for cairo
 License:        (LGPLv2 OR MPLv1.1) AND MIT AND Public Domain
 
-Requires:       %{name} = %{version}-%{release}
-Requires:       fontconfig-devel
-Requires:       freetype-devel
-Requires:       libpng-devel
-Requires:       pixman-devel
```

Comment:
Why do we no longer require these and other run-time dependencies?

Reply:
Added back!